crates/floresta-wire/src/p2p_wire/node.rs

```diff
@@ -386,24 +548,20 @@ where
                 self.handle_get_peer_info(responder);
                 return;
             }
-            UserRequest::Connect((addr, port)) => {
-                let addr_v2 = match addr {
-                    IpAddr::V4(addr) => AddrV2::Ipv4(addr),
-                    IpAddr::V6(addr) => AddrV2::Ipv6(addr),
-                };
-                let local_addr = LocalAddress::new(
-                    addr_v2,
-                    0,
-                    AddressState::NeverTried,
-                    0.into(),
-                    port,
-                    self.peer_id_count as usize,
-                );
-                self.open_connection(ConnectionKind::Regular(ServiceFlags::NONE), 0, local_addr)
+            UserRequest::Add((addr, port, v2transport)) => {
+                let _ = self
```

Comment:
Here, instead of ignoring the error you would handle it and return `true/false` given the response.